### ISSUE TITLE
3.0.98 will cause a crash.

### ISSUE THREAD

**Issue by MagmaBro123** [bernie-g/geckolib#316]
Title. Report below, modlist inside it as well. 

GekoLib is mentioned a lot throughout it, so I doubt its other mods, but I may be wrong.

https://pastebin.com/Tz5Am3cg

**Comment by AzureDoom** [comment]
Fixed #319 merged, will be released shortly.

**Comment by MagmaBro123** [comment]
Thanks.

**Comment by AzureDoom** [comment]
> Thanks. I'm not using Fabric, so .99 isn't useful to me atm, unless Forge is still taking a while.

it's all still building and pending on CurseForge

**Comment by MagmaBro123** [comment]
Ah, alright, sorry.

**Comment by Abalieno** [comment]
I think I'm still seeing this on 3.0.71 fabric 1.18.2

**Comment by AzureDoom** [comment]
> I think I'm still seeing this on 3.0.71 fabric 1.18.2 (3.0.68 loads fine and the problem started in 3.0.70)

New issue for different versions and include logs
